### PR TITLE
Add SignAndSend command

### PR DESCRIPTION
**Pull request by leoneparise**

### CODE REVIEW

File: TrustSDK/Classes/Client/Types/TrustCommand.swift [hunk 1]
```diff
@@ -10,11 +10,73 @@ import TrustWalletCore
 enum CommandName: String {
     case getAccounts = "sdk_get_accounts"
     case sign = "sdk_sign"
```
Comment (vikmeup) [inline TrustSDK/Classes/Client/Types/TrustCommand.swift]: can we use `sdk_sign` but make optional param as `send: Bool` here? i don't think need comment is needed here

File: TrustSDK/Classes/Client/Types/TrustCommand.swift [hunk 2]
```diff
@@ -10,11 +10,73 @@ import TrustWalletCore
...
+
+enum SignMetadataName: String {
+    case dApp = "dapp"
 }
 
 public extension TrustSDK {
+    enum SignMetadata {
+        case dApp(name: String, url: URL?)
+
+        init?(value: String) {
```
Comment (vikmeup) [inline TrustSDK/Classes/Client/Types/TrustCommand.swift]: what does code do?

Comment (leoneparise) [reply TrustSDK/Classes/Client/Types/TrustCommand.swift]: I added a generic code to encode/decode multilevel dictionaries into `[URLQueryItem]`. Please check `DictionaryTests` to understand how it works.

File: TrustSDK/Classes/Client/Signer/Signer.swift [hunk 3]
```diff
@@ -15,18 +15,32 @@ public extension TrustSDK {
...
+            do {
+                let command: TrustSDK.Command = .sign(coin: coin, input: try input.serializedData(), send: false, metadata: metadata)
                 try TrustSDK.send(request: SignRequest(command: command, callback: callback))
             } catch {
                 callback(Result.failure(error))
             }
         }
+
+        public func signThenSend(input: SigningInput, metadata: SignMetadata? = nil, callback: @escaping ((Result<String, Error>) -> Void)) {
+            if !TrustSDK.isSupported(coin: coin) {
```
Comment (ArtemGoryunov) [inline TrustSDK/Classes/Client/Signer/Signer.swift]: guard?

**Pull request merged**